guides/common/modules/proc_enabling-openscap.adoc

```diff
@@ -12,3 +12,8 @@ To use the OpenSCAP plug-in and content on an external {SmartProxy}, you must en
 # {installer-scenario-smartproxy} \
 --enable-foreman-proxy-plugin-openscap
 ----
++
+Append one of the following arguments to the command:
```

Comment:
The current phrasing implies (to me) they're mutually exclusive but it's also safe to use both at the same time. It may be redundant in most cases, but I can imagine some servers are managed by Puppet and some by Ansible. Having both installed kinds of modules at the same time doesn't hurt.